src/components/Tooltip/TooltipOverlay.js

```diff
@@ -0,0 +1,90 @@
+/* eslint-disable */
+import React, { Component, Fragment } from 'react';
+import PropTypes from 'prop-types';
+import { Portal } from '../Portal';
+import { debounce } from '../../helpers/debounce';
+import { getWindowRect, getRectForNode } from '../../helpers/geometry';
+import { WindowEvent } from '../WindowEvent';
+import styles from './TooltipOverlay.module.scss';
+
+class TooltipOverlay extends Component {
+  static displayName = 'TooltipOverlay';
+
+  static defaultProps = {
+    eventDebounce: 300
+  }
+
+  static propTypes = {
+    activator: PropTypes.func.isRequired,
+    overlay: PropTypes.func.isRequired,
+    eventDebounce: PropTypes.number
+  }
+
+  state = {
+    mounted: false,
+    position: {},
+    preferredDirection: {
+      top: null,
+      bottom: null,
+      left: null,
+      right: null
+    }
+  }
+
+  componentDidUpdate() {
+    // Check if the activator has been mounted
+    // then calculate measurements, undebounced
+    if (!this.state.mounted && !Object.keys(getRectForNode(this.activator)).length) {
```

Comment:
Bob found an issue where some tooltips would not get position unless window was scrolled or resized.

I initially had the first `handleMeasurement` in cDM, but that did not guarantee the activator component is rendered. Not sure if there's a better way to do this.